PROTOCOL.md

```diff
@@ -424,6 +429,67 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests are sent with `REQUESTID`. `REQUESTID` is used to identify which
+action the request was intended for and is used to protect against race
+conditions with the cancel button. The first decision of a match has a
+`REQUESTID` of 1, and each successful decision increments the counter by 1.
+`REQUESTID` is optional.
```

Comment:
Isn't `|/undo` not sent with `|REQUESTID`? If so would it ignore it if you sent `|REQUESTID` anyways? I don't wan to say "all of the following commands" if that creates confusion with how `|/undo` works.